indico/web/client/js/react/forms/index.js

```diff
@@ -14,6 +14,7 @@ export {
   FinalRadio,
   FinalSubmitButton,
   FinalTextArea,
+  unsortedArraysEqual,
```

Comment:
I think there's at least one place importing it directly from `indico/react/forms/fields`; maybe you could update that place to import it from here instead?